content/millennium/r4/base/individuals/patient.md

```diff
@@ -51,6 +60,7 @@ Search for Patients that meet supplied query parameters:
 _Implementation Notes_
 
 * Direct secure email will not be returned.
+* Usual will be represented by [Identifier.use](https://hl7.org/fhir/R4/datatypes-definitions.html#Identifier.use){:target="_blank"}
```

Comment:
Looks like I got interrupted yesterday and didn't get this comment posted yesterday. The wording here and line 149 is a bit backward. "Usual" represents the identifier's use, not the other way around.

Perhaps:
```suggestion
* [Patient.identifier.use](https://hl7.org/fhir/R4/datatypes-definitions.html#Identifier.use){:target="_blank"} will have a value of `usual`
```